src/redux/thunks/loadKubeConfig.ts

```diff
@@ -1,17 +1,15 @@
 import * as k8s from '@kubernetes/client-node';
 
-import {AnyAction} from '@reduxjs/toolkit';
-
 import fs from 'fs';
 import log from 'loglevel';
 
 import {AlertEnum} from '@models/alert';
-import {KubeConfig, KubeConfigContext} from '@models/kubeConfig';
+import {KubeConfig, KubeConfigContext} from '@models/appconfig';
 
 import {setAlert} from '@redux/reducers/alert';
-import {setContexts} from '@redux/reducers/appConfig';
+import {updateProjectKubeConfig} from '@redux/reducers/appConfig';
 
-export const loadContexts = async (configPath: string, dispatch: (action: AnyAction) => void) => {
+export const loadContexts = async (configPath: string, dispatch: any, currentContext?: string) => {
```

Comment:
any reason for removing the `(action: AnyAction) => void` type from `dispatch`?

Reply:
beware that you cannot dispatch any Thunks from the main process, only plain actions